src/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -113,18 +115,13 @@ public class DataController : ControllerBase
             [FromRoute] Guid instanceGuid,
             [FromQuery] string dataType)
         {
-            if (string.IsNullOrWhiteSpace(dataType))
-            {
-                return BadRequest("Element type must be provided.");
-            }
-
```

Comment:
No, the addition `#nullable enable` on line 1 will trigger asp net to return `BadRequest` with a `Problem` json structure that explains that `dataType` is a required part of the query string, because it isn't nullable.

Reply:
Have you tested this? Might be som error in my test setup, but when I tested it locally it didn't return bad request with Problem json when I omitted the dataType param

Reply:
Hmm... Not here specifically. I got that behaviour (breaking a test) when running tests under net8.0 with `<nullable>enabled</nullable>` in project.json. Therefore I was very careful with [FromQuery] parameters that was not nullable. Maybe it's only when nullability is enabled globally, maybe only net8.0?

Reply:
You were right. The difference was that I tested this with `OptionsController` and it was annotated with `[ApiController]` which causes automatic BadRequest for non nullable query strings when nullable is enabled. I wrote a test for this "BadRequest" response and added the annotation to make the test work in ab43649f.